README.md

```diff
@@ -1,2 +1,32 @@
 # supertokens-web-js (In progress)
+
 SuperTokens SDK for vanilla JS for all recipes
+
```

Comment:
Should all be supertokens.com, not supertokens.io in this. I am referring to all the links